.gitignore

```diff
@@ -4,5 +4,5 @@
 uploads
 node_modules
 .idea
-config/config*.env
```

Comment:
please keep this, I'm playing around with a few different config options and don't want to accidentally submit a production config